WcaOnRails/app/views/shared/_associated_events_picker.html.erb

```diff
@@ -23,7 +23,7 @@ Note: form_builder.object is the object having associated events.
 
 <div class="form-group <%= "has-error" unless errors.empty? %>">
   <%= label_tag events_association_name, class: "associated-events-label" do %>
-    <%= label %>
+    <%= label %> (<span class="events-selected-count"></span> selected)
```

Comment:
Yeah, it's problematic for Polish and the same goes for some others I guess.
I'd suggest doing it this way

```erb
(<%= t 'path.to.selected_count' %>: <span class="events-selected-count"></span>)
```

```yml
en:
  path:
    to:
      selected_count: "selected count"
```
I think this one should be count-independent (at least for Polish and English for sure ^^).